src/main/java/net/mcreator/ui/modgui/VillagerProfessionGUI.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.modgui;
+
+import net.mcreator.element.parts.MItemBlock;
+import net.mcreator.element.types.VillagerProfession;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.SearchableComboBox;
+import net.mcreator.ui.component.util.ComboBoxUtil;
+import net.mcreator.ui.component.util.ComponentUtils;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.dialogs.TextureImportDialogs;
+import net.mcreator.ui.help.HelpUtils;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.ui.laf.renderer.WTextureComboBoxRenderer;
+import net.mcreator.ui.minecraft.MCItemHolder;
+import net.mcreator.ui.minecraft.SoundSelector;
+import net.mcreator.ui.validation.AggregatedValidationResult;
+import net.mcreator.ui.validation.ValidationGroup;
+import net.mcreator.ui.validation.Validator;
+import net.mcreator.ui.validation.component.VComboBox;
+import net.mcreator.ui.validation.component.VTextField;
+import net.mcreator.ui.validation.validators.TextFieldValidator;
+import net.mcreator.ui.validation.validators.UniqueNameValidator;
+import net.mcreator.ui.workspace.resources.TextureType;
+import net.mcreator.util.ListUtils;
+import net.mcreator.util.StringUtils;
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.File;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+public class VillagerProfessionGUI extends ModElementGUI<VillagerProfession> {
+
+	private final VTextField displayName = new VTextField(30);
+	private final MCItemHolder pointOfInterest = new MCItemHolder(mcreator, ElementUtil::loadBlocks);
+	private final SoundSelector actionSound = new SoundSelector(mcreator);
+	private final JComboBox<String> hat = new JComboBox<>(new String[] { "None", "Partial", "Full" });
```

Comment:
I guess we should either uppercase the values here (which is fine as we lowercase them in code anyway) or modify help tips to use these values instead of uppercased ones.

Reply:
Done, I lowercased the values in the help tips